static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -98,28 +98,33 @@ function BaseDraggableTabList({
         <TabListWrap {...tabListProps} className={className} ref={tabListRef}>
           {persistentTabs.map(item => (
             <Fragment key={item.key}>
-              <Reorder.Item
-                key={item.key}
-                value={item}
-                style={{display: 'flex', flexDirection: 'row'}}
-                as="div"
-                dragConstraints={tabListRef} // Sets the container that the tabs can be dragged within
-                dragElastic={0} // Prevents tabs from being dragged outside of the tab bar
-                dragTransition={{bounceStiffness: 400, bounceDamping: 40}} // Recovers spring behavior thats lost when using dragElastic
-                layout="position"
-              >
-                <Tab
+              <TabItemWrap isSelected={state.selectedKey === item.key}>
```

Comment:
Ideally we could just apply the z-index styles directly to the `<Reorder.Item>`, but doing this results in the z-index set to `unset`, which fails to solve the bug. This is intentional, and is known to cause this bug: https://github.com/framer/motion/pull/1367